example.env

```diff
@@ -10,4 +10,5 @@ PG_PASSWORD=postgres
 PG_HOST=localhost
 PG_PORT=5432
 PG_DB=jobs
-STATIC_FILES_LOCATION=REALTIVE_TO_SERVER DIR
\ No newline at end of file
+STATIC_FILES_LOCATION=REALTIVE_TO_SERVER DIR
```

Comment:
A suggestion: `STATIC_FILES_LOCATION=./src/build # RELATIVE_TO_SERVER DIR` 

Comments in `.env` are done with `#` rather than `//`